packages/node_modules/@webex/plugin-messages/src/messages.js

```diff
@@ -54,6 +54,11 @@ const Messages = WebexPlugin.extend({
    * This is an alternate approach to registering for messages webhooks.
    * The events passed to any registered handlers will be similar to the webhook JSON,
    * but will omit webhook specific fields such as name, secret, url, etc.
+   * The messages.listen() event objects can also include additional fields not
+   * available in webhook JSON. These fields are text, markdown, and files.
+   * These fields are available when their details are included in the WebSocket's
+   * activity object. Retrieving other fields, such as the html field,
```

Comment:
\`activity\` object. Retrieving other fields, such as the \`html\` field,

Reply:
I have addressed this change.